manifests/plugin/salt.pp

```diff
@@ -58,6 +58,7 @@
     ensure  => file,
     content => template('foreman_proxy/plugin/salt_master.conf.erb'),
     owner   => 'root',
+    group   => 'foreman-proxy',
```

Comment:
I'm just wondering - we already have a parameter named `$user`. Would it make sense to introduce `$group` parameter here? It could even be `Optional[String[1]] $group = undef` and then default to the value of `$user`:
```suggestion
    group   => pick($group, $user),
```

Reply:
Sounds useful to me - where would you add the `Optional[String[1]] $group = undef` group creation?

Reply:
Just below `... $user = ...` in the class definition.

Reply:
How can the `$group` param be set/actually used if we initialise it to `undef`?
Is the default value of `$user = foreman-proxy`?

Reply:
[pick()](https://forge.puppet.com/modules/puppetlabs/stdlib/reference#pick) does that for us here. We don't set `$group`, but rather fall back `$user` if `$group` is unset. Using `undef` by default has the benefit that the installer doesn't store the concrete answer.

Reply:
Alright, thanks!